Makefile

```diff
@@ -51,4 +52,7 @@ push:
 partition-gpu:
 	docker build --pull -t ${REGISTRY}/${PARTITION_GPU_IMAGE}:${TAG} -f partition_gpu/Dockerfile .
 
+fastsocket_installer:
+	docker build --pull -t ${REGISTRY}/${FASTSOCKET_INSTALLER_IMAGE}:${TAG} -f fast-socket-installer/Dockerfile .
```

Comment:
Should we use a different tag here?

Reply:
I put the tag here for testing purpose. I would like to see how much difference from 'partition-gpu'. I will change the tag later after the louhi flow works.